tasks/setup.yml

```diff
@@ -27,6 +27,7 @@
     owner: "{{ sqitch_migrations_system_user }}"
     group: "{{ sqitch_migrations_system_group }}"
     path: "{{ item }}"
+    mode: '0644'
```

Comment:
A bit nit-picky but should we use double-quotation marks here (same as in line 43)?

Reply:
Yes will do